app/src/main/java/com/mensinator/app/CalendarScreen.kt

```diff
@@ -90,6 +91,10 @@ fun CalendarScreen() {
     var previousFirstPeriodDate by remember { mutableStateOf<LocalDate?>(null) }
     val colorMap = ColorSource.getColorMap(isDarkMode())
 
+    // Trigger notification with custom message
+    val initPeriodKeyOrCustomMessage = dbHelper.getStringSettingByKey("period_notification_message")
```

Comment:
Use `StringSetting.PERIOD_NOTIFICATION_MESSAGE.settingDbKey` here? Not sure

Reply:
I think that's a fair change, will put it in

Reply:
changed